src/build.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LearderBoard</title>
```

Comment:
- [ ] I can see that you added styling to your app but you forgot to import it in your HTML, please do so to match the wireframe provided by Microverse.